tests/general/test_lock.py

```diff
@@ -8,51 +8,39 @@
 from inbox.util.file import Lock
 
 
-def lock(block, filename=None):
+def tmp_lock(block, filename=None):
     if filename is None:
         handle, filename = tempfile.mkstemp()
     return Lock(filename, block=block)
 
 
-@pytest.fixture
```

Comment:
This fixtures are so simple and only used once that not having them makes code more readable